docs/ble.md

```diff
@@ -118,97 +133,114 @@ one previously sent with the 'accept' event.
 Emitted with 'poweredOn' when the BLE stack is ready to be used. No other states
 are supported at this time.
 
-### BLE.disconnect
-
-`void disconnect(string address);`
+BLE API
+-------
+### ble.disconnect(address)
+*'address' *string* The address of the connected client.
 
 Disconnect the remote client.
 
-The `address` is the address of the connected client.
+### ble.startAdvertising(name, uuids, url)
+* name *string* The `name` is limited to 26 characters and will be
+  advertised as the device name to nearby BLE devices.
+* uuids *string[]*  The `uuids` array may contain at most 7 16-bit
+  UUIDs (four hex digits each).  These UUIDs identify available
+  services to nearby BLE devices.
+* url *string* The `url` is optional and limited to around 24
+  characters (slightly more if part of the URL is able to be
+  [encoded](https://github.com/google/eddystone/tree/master/eddystone-url). If
+  provided, this will be used to create a physical web advertisement
+  that will direct users to the given URL. At that URL they might be
+  able to interact with the advertising device somehow.
+  
+Advertises the name of the device.
```

Comment:
name and **url**

url is really the interesting bit

Reply:
Done.